app/models/post.rb

```diff
@@ -0,0 +1,20 @@
+class Post < ApplicationRecord
+  validates :title, presence: true
+
+  belongs_to :users
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Here Post belongs_to one user at a time not many users.

Optional: You can also look at how to use `class_name`